applications/fluid_structure_interaction/cylinder_with_flag/input.json

```diff
@@ -23,6 +23,6 @@
     "Output": {
         "OutputDirectory": "output/cylinder_with_flag/",
         "OutputName": "test",
-        "WriteOutput": "false"
+        "WriteOutput": "true"
```

Comment:
just a minor comment here: this change is probably not intended for the present PR